src/components/NavigationDesktop/NavigationItemDesktop.js

```diff
@@ -143,7 +145,7 @@ class NavigationItemDesktop extends Component {
 
     return (
       <Fragment>
-        <Button className={primaryNavItem} color="inherit" onClick={this.onClick}>
+        <Button className={primaryNavItem} color="inherit" onClick={this.onClick} href={`${this.linkPath(navItem)}`}>
```

Comment:
href={`${this.linkPath(navItem)}`} -> `href={this.linkPath(navItem)}` since it's already a string

Reply:
@mikemurray updated